src/client/driver/driver.js

```diff
@@ -1459,7 +1523,7 @@ export default class Driver extends serviceUtils.EventEmitter {
     }
 
     _initParentWindowLink () {
-        if (window.opener)
+        if (window.opener && window.opener !== window && this.windowId)
```

Comment:
Add comment describing this condition `window.opener !== window && this.windowId`;

Reply:
done